src/coreclr/inc/cordebug.idl

```diff
@@ -3988,14 +3988,14 @@ interface ICorDebugRegisterSet : IUnknown
         REGISTER_LOONGARCH64_F31,
 
         REGISTER_RISCV64_PC = 0,
-        REGISTER_RISCV64_RA,
         REGISTER_RISCV64_SP,
+        REGISTER_RISCV64_FP,
```

Comment:
Why is this reordering needed?

Reply:
In case of register indexes, we have "fixed" indexes for PS, SP and FP:
https://github.com/dotnet/runtime/blob/173eb1fb769edd42c856a83ce972ba0a63b425ca/src/coreclr/pal/prebuilt/inc/cordebug.h#L9476-L9478
in the same time, code also count on this and don't use arch specific registers by unique names for PS, SP and FP. For example:
https://github.com/dotnet/runtime/blob/173eb1fb769edd42c856a83ce972ba0a63b425ca/src/coreclr/debug/di/rsthread.cpp#L6135-L6143